crowbar_framework/lib/barclamp_catalog.rb

```diff
@@ -64,12 +64,10 @@ def category(barclamp)
   end
 
   def display_name(barclamp)
-    display = barclamps[barclamp]['display']
-
-    if display.nil? or display.empty?
-      barclamp.titlecase
+    if barclamps[barclamp]
+      display = barclamps[barclamp]['display'] || barclamp.titlecase
```

Comment:
Useless assignment to variable - `display`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.